### PR TITLE
Logo for Music Player

### PR DESCRIPTION
**Pull request by michaelizer**
Sorry for the "," spam

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 [![Android Sweets #27](https://img.shields.io/badge/Android%20Sweets-%2327-ff69b4.svg?style=true)](https://androidsweets.ongoodbits.com/2016/07/14/issue-27)
 [![Awesome Android #22](https://img.shields.io/badge/Awesome%20Android-%2322-green.svg?style=true)](https://android.libhunt.com/newsletter/22)
 
-![Icon](https://raw.githubusercontent.com/andremion/Music-Player/master/app/src/main/res/mipmap-hdpi/ic_launcher.png)
+<p align="center"><img src="app/src/main/res/logo-mipmap-xxxhdpi/ic_launcher.png" alt="Music Player" height="200px"></p>
```
Comment (andremion) [inline README.md]: You don't need to change this. Since the ic_launcher file if updated, it is done.

**Comment by andremion** [comment]
Hi @michaelizer, sorry the delayed response.
I am currently reviewing your PR...

**Comment by andremion** [comment]
We need to fix this commit history to have just one commit with your changes

File: README.md [hunk 2]
```diff
@@ -7,7 +7,7 @@
 [![Android Sweets #27](https://img.shields.io/badge/Android%20Sweets-%2327-ff69b4.svg?style=true)](https://androidsweets.ongoodbits.com/2016/07/14/issue-27)
 [![Awesome Android #22](https://img.shields.io/badge/Awesome%20Android-%2322-green.svg?style=true)](https://android.libhunt.com/newsletter/22)
 
-![Icon](https://raw.githubusercontent.com/andremion/Music-Player/master/app/src/main/res/mipmap-hdpi/ic_launcher.png)
```
Comment (andremion) [inline README.md]: You don't need to create another folder named as `logo`. You can use all `mipmap` folder already created.

**Comment by michaelizer** [comment]
@andremion restored the readme and uploaded the ic_launcher_foreground.png in mipmap-xxxhdpi. 

Even tho I'm looking how to delete commits, it's still away from my understanding haha. Could you please delete the folders I wrongly created?

**Comment by andremion** [comment]
I can't delete your commits because it is from your fork. Since you are using the web to make the changes you are limited with the git commands.

I think you could start it again.

Please, open another pull request just adding the `mipmap-xxxhdpi/ic_launcher_foreground.png` file. I will do the rest... including mentioning you in `README` file...

After that we can think in changing some assets for the play store.

**Pull request closed without merging**